spec/models/validations/sales/sale_information_validations_spec.rb

```diff
@@ -505,6 +505,94 @@
     end
   end
 
+  describe "#validate_outright_sale_value_matches_mortgage_plus_deposit" do
+    context "with a 2024 outright sale log" do
+      let(:record) { FactoryBot.build(:sales_log, value: 300_000, ownershipsch: 3, saledate: Time.zone.local(2024, 5, 1)) }
+
+      context "when a mortgage is used" do
```

Comment:
I've not got tests for not applying when this is false, because in that case mortgage and deposit aren't routed to.

Reply:
Would it make sense to add a test for BU? What would happen if they mortgage used is no in BU but the mortgage values are still given? I don't think it would trigger, but might be worth double checking

Reply:
The mortgage value ends up being cleared in that case (and this validation doesn't trigger, which it shouldn't).